### PR TITLE
Issue #19: Add meta tags for better link preview

### PR DESCRIPTION
**Pull request by fiigmnt**
preview

### CODE REVIEW

File: web-previewer/index.html [hunk 1]
```diff
@@ -6,6 +6,26 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
     <link rel="stylesheet" href="https://rsms.me/inter/inter.css">
     <title>Developer DAO: Avatar Preview</title>
```
Comment (Dhaiwat10) [inline web-previewer/index.html:8]:
```suggestion
```

We need to remove this old `<title>` tag! 😄

**Review by Dhaiwat10 — CHANGES_REQUESTED** [review]
Looking good. Thanks for putting this in! Suggested just one small change.

File: web-previewer/index.html (hunk 1 above)
Comment (fiigmnt) [reply web-previewer/index.html:8]: ha! good call, sorry I missed that one. 👍

**Comment by Dhaiwat10** [comment]
Awesome! Just tweaked some wording here & there. We're good to go.

**Pull request merged**